src/App.js

```diff
@@ -1,19 +1,46 @@
 import { Component } from 'react';
-import logo from './react-logo.svg';
 import './App.css';
 import React from 'react';
+import request from 'superagent';
+import Header from './Header';
+import PokeList from './PokeList';
+import Footer from './Footer';
+import PokeSearch from './PokeSearch';
 
+const POKEDEX_API_URL = 'https://pokedex-alchemy.herokuapp.com/api/pokedex';
 class App extends Component {
 
+  state = {
+    pokemon: []
+  }
+
+  async componentDidMount() {
+    this.handleSearch();
+    // const response = await request.get(POKEDEX_API_URL);
+    // this.setState({ pokemon: response.body.results });
+  }
+
+  handleSearch = async (search) => {
+    const response = await request(POKEDEX_API_URL)
+      .query({ pokemon: search });
```

Comment:
you'll also need to initialize your sort state above and query the fetch here. like `.query({ sort: sortState })`